src/service.rs

```diff
@@ -1,6 +1,6 @@
 use futures::{prelude::*, sync::mpsc};
 use log::{debug, error, trace, warn};
-use std::collections::{HashMap, HashSet, VecDeque};
+use std::collections::{vec_deque::VecDeque, HashMap, HashSet};
```

Comment:
why?

Reply:
It's not an important reason, just since IDE finds `std::collections::vec_deque::VecDeque` but cannot find `std::collections::VecDeque`. So I change it handily.

Reply:
You can throw your IDE 😂